src/olympia/shelves/serializers.py

```diff
@@ -134,10 +150,9 @@ def get_addons(self, obj):
         return addons
 
 
-class ShelfEditorialSerializer(ShelfSerializer):
+class ShelfEditorialSerializer(serializers.ModelSerializer):
     title = serializers.CharField()
     footer_text = serializers.CharField()
-    addons = None  # we don't the results for this serializer
```

Comment:
Why is this not needed anymore?

Reply:
ShelfEditorialSerializer now isn't subclassing another serializer that had defined an `addons` field.